### PR TITLE
remove working dir

### PR DESCRIPTION
**Pull request by Krigpl**
It's useless.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -30,7 +30,7 @@ eth-tester==0.1.0-beta.24
 eth-utils==1.0.3
 ethereum==1.6.1
 Golem-Messages==2.12.2
-Golem-Verificator==1.3.0
+-e git://github.com/golemfactory/golem-verificator.git@84b71eab899b7831786e642ca407a04138aca802#egg=golem_verificator
```
Comment (maaktweluit) [inline requirements.txt]: is this update related to the cleanup? else a separate PR would be nice

Should be released before merge

File: golem/task/taskthread.py [hunk 2]
```diff
@@ -122,9 +120,6 @@ def __do_work(self):
         abs_res_path = os.path.abspath(os.path.normpath(self.res_path))
         abs_tmp_path = os.path.abspath(os.path.normpath(self.tmp_path))
 
-        self.prev_working_directory = os.getcwd()
-        os.chdir(os.path.join(abs_res_path,
```
Comment (maaktweluit) [inline golem/task/taskthread.py:126]: where will taskthread work now there is no `chdir`?

**Review by maaktweluit — CHANGES_REQUESTED** [review]
LGTM!

2 small comments, will approve on reply or resolve :)

File: requirements.txt (hunk 1 above)
Comment (maaktweluit) [reply requirements.txt]: should have checked the linked PR, is related :)

File: golem/task/taskthread.py (hunk 2 above)
Comment (Krigpl) [reply golem/task/taskthread.py:126]: Hmmmmmm, good catch. Maybe it wasn't completely useless. Although I'm pretty sure working directory here could be simply set to `/tmp/<whatever>`, right?

Comment (maaktweluit) [reply golem/task/taskthread.py:126]: haha, almost useless still :P

yes, that would work

i also see a tmp_path variable available, maybe that is a good working dir to `chdir` to?

Comment (Elfoniok) [reply golem/task/taskthread.py:126]: /tmp/ ?  I think its always good to remove chdir (like who use that?!, what if separate threads are saving files and someone just change its working dir?). I am pretty sure that this is working dir for local test, during adding task (but better confirm this) so datadir/network_name/tmp dir ?

**Review by Elfoniok — APPROVED** [review]

File: golem/task/taskthread.py (hunk 2 above)
Comment (Elfoniok) [reply golem/task/taskthread.py:126]: Hey i checked this, so this is base class and its never used (i tried as provider and requestor), derived class which is DockerTaskThread is used all the time and currently i don't see when one could want to use base implementation.

**Pull request merged**